REQUIRE

```diff
@@ -3,3 +3,4 @@ Polynomials 0.1.0
 Reexport
 SpecialFunctions
 FFTW
+Compat 1.2
```

Comment:
```suggestion
Compat 1.3
```
Relevant change is
https://github.com/JuliaLang/Compat.jl/commit/052636a58dcffc3c810d48bc3818293dde924902

Reply:
Thanks for catching that, and sorry that my PRs have been a little sloppy! I should really be working on other stuff at the moment, so I might be in a bit too much of a rush to finish this stuff up.

Reply:
Sorry if my comments are rather terse and might therefore sound a bit harsh sometimes. I should also be doing other stuff...

Reply:
Not at all, they are appreciated! At least I'm in good company while not doing what I really should be doing.